README.md

```diff
@@ -1,13 +1,133 @@
 # Bias Calculator Project
```

Comment:
Might be better to rename this to Bias Visualizer Project, what do you think?

Reply:
I agree on this